global.json

```diff
@@ -1,13 +1,13 @@
 {
   "sdk": {
-    "version": "5.0.406",
+    "version": "6.0.202",
```

Comment:
I don't know which arcade version flows into arcade-services. Should we try to update the subscription to the release/6.0 branch? @mmitche @riarenas

Reply:
(after this PR goes in, of course)

Reply:
No version flows into here (or into helix-services), to my knowledge. Azure services and arcade aren't... really friends. Arcade is really focused on products that release big stable packages once in a blue moon, not services that ship via special deployment instructions in code and don't actually produce any binaries... it's been a struggle.